pkg/common/resources/resources.go

```diff
@@ -403,6 +403,13 @@ func FitIn(larger, smaller *Resource) bool {
 	return larger.fitIn(smaller, false)
 }
 
+// Check if smaller fits in larger
+// Types not defined in the larger resource are considered 0 values for Quantity
+// A nil resource is treated as max resource
+func FitInUndef(larger, smaller *Resource) bool {
```

Comment:
This is the same as calling `larger.FitInMaxUndef(smaller)` I don't think we need a new function for that.